src/app/board/board.component.scss

```diff
@@ -6,20 +6,21 @@
   margin: em(20) auto;
   padding-top: em(96);
   display: flex;
-  background-color: rgba($color-pf-black-100, 0.2);
+  background-color: $color-pf-white;
```

Comment:
I believe the previous one used to make the background transparent. Will it work the same way now?